commands/types/fakedoor/message.go

```diff
@@ -0,0 +1,42 @@
+package types
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/fatih/color"
+)
+
+var message = "The support for %s hasn't been implemented yet.\nIf you would like us to implement it, please attend a poll by at:\n%v and give a feedback.\nWe're happy to implement it quickly based on demand!\n"
+var IssueURL = "https://docs.google.com/forms/d/e/1FAIpQLSfraK3Mu0u96rAau_OO-heoHE_z7gCZwX8Dw034RlzYf27M1Q/viewform?usp=sf_link"
+
+type ColoredString struct {
+	Word  string
+	Color *color.Color
+}
+
+func (p ColoredString) String() string {
+	if p.Word == "" {
+		log.Panicln("word cannot be empty")
+	}
+	return p.Color.Sprint(p.Word)
+}
+
+func DecorateStringColorFgCyanWithUnderline(str string) *ColoredString {
+	if str == "" {
+		log.Panicln("given string cannot be empty")
+	}
+	return &ColoredString{
+		Word:  str,
+		Color: color.New(color.FgCyan).Add(color.Underline),
+	}
+}
+
```

Comment:
Let's remove the color stuff for now.

Reply:
Can you explain why ? Because of I exploited extra library other than `go-colorable` ?

Reply:
The requirements don't include the color and styling. That's something we should decide. Let's not add extra dependencies without discussing about them first.

Reply:
Fixed: [a3c9d21](https://github.com/hazelcast/hazelcast-commandline-client/pull/45/commits/a3c9d213edcf6740fd480352948d34aaa5e57b3d).